addons/web_calendar/static/src/js/web_calendar.js

```diff
@@ -21,13 +21,23 @@ var QWeb = core.qweb;
 function get_fc_defaultOptions() {
     var dateFormat = time.strftime_to_moment_format(_t.database.parameters.date_format);
 
+    // moment.js converts '%p' to 'A' for 'AM/PM'
+    // But FullCalendar v1.6.4 supports 'TT' format for 'AM/PM' but not 'A'
+    var timeFormat = time.strftime_to_moment_format(_t.database.parameters.time_format).replace('A', 'TT');
+
     // adapt format for fullcalendar v1.
     // see http://fullcalendar.io/docs1/utilities/formatDate/
     var conversions = [['YYYY', 'yyyy'], ['YY', 'y'], ['DDDD', 'dddd'], ['DD', 'dd']];
     _.each(conversions, function(conv) {
         dateFormat = dateFormat.replace(conv[0], conv[1]);
     });
 
+    var hourFormat = function(timeFormat){
+        if (/H/.test(timeFormat))
+            return 'HH:mm';
+        return 'hh TT';
```

Comment:
comment with meaning of the syntax http://fullcalendar.io/docs1/utilities/formatDate/